ReportUtilityBills/tests/Sample Tiered Rate.json

```diff
@@ -0,0 +1,659 @@
+{
+  "items": [
+    {
+      "name": "Sample Tiered Rate",
+      "sector": "Residential",
+      "description": "Sample file defined by BEopt.",
+      "fixedmonthlycharge": 9,
+      "fixedchargeunits": "$/month",
+      "flatratebuy": 0,
+      "flatratefueladj": 0,
```

Comment:
I'm confused by some of the fields in these sample files. For example, these three fields aren't in the [documentation](https://openei.org/services/doc/rest/util_rates/?version=3#response-fields), so why are they here? We need to make these sample files consistent with version 3 rates from URDB.

Does your code actually use these fields?

Reply:
I believe it uses `flatratebuy` and `flatratefueladj`.

Reply:
I removed `flatratebuy`, `flatratefueladj`, and `fixedchargeunits` from the "Sample *.json" test files. I updated the `self.detailed_electric` method to no longer use `flatratebuy` or `flatratefueladj`.